iCub_manual/ros2/README.md

```diff
@@ -13,7 +13,7 @@ Right now are published on ROS2 topics:
 
 The steps to run rviz visualization correctly are:
 
-- Edit `iCubGazeboV3/model.urdf` putting  [`icub_ROS2.xml`](https://github.com/robotology/robots-configuration/blob/devel/iCub/conf_icub3/wrappers/icub_ROS2.xml) `robotinterface` xml file used at startup.
+- Edit `iCubGazeboV*/model.urdf` putting  [`icub_ROS2.xml`](https://github.com/robotology/robots-configuration/blob/devel/iCub/conf_icub3/wrappers/icub_ROS2.xml) `robotinterface` xml file used at startup.
```

Comment:
This file was removed in https://github.com/robotology/icub-models-generator/pull/281 .

Reply:
Mmm if I recall correctly the `_manual` folder are not in `icub-models-generator`

Reply:
Actually the README links a file in robots-configuration? Anyhow the link to file https://github.com/robotology/robots-configuration/blob/devel/iCub/conf_icub3/wrappers/icub_ROS2.xml is broken.

Reply:
At a first glance, the `icub_ROS2.xml` file is only present for the `iCubGazeboV2_5_visuomanip` model, see https://github.com/robotology/icub-models/blob/63f8cee83da547b29f24c9daad6cab41077acb32/iCub_manual/conf_manual/iCubGazeboV2_5_visuomanip/icub_ros2.xml , while there is no similar file for non-visuomanip models: https://github.com/robotology/icub-models/blob/63f8cee83da547b29f24c9daad6cab41077acb32/iCub/conf/icub.xml .

To keep things simple, should we only document this for `iCubGazeboV2_5_visuomanip`  for now?

Reply:
I removed the link, we can always cleanup the situation later.